clusters/app.ci/prow-monitoring/mixins/_prometheus/dptp_alerts.libsonnet

```diff
@@ -160,6 +160,33 @@
           }
         ],
       },
+      {
+        name: 'openshift-priv-image-building-jobs-failing',
+        rules: [
+          {
+            alert: 'openshift-priv-image-building-jobs-failing',
+            expr: |||
+             (
+               sum(
+                 rate(prowjob_state_transitions{job="prow-controller-manager",job_name=~".*-images",org="openshift-priv",state="failure"}[6h])
+               )
+               /
+               sum(
+                 rate(prowjob_state_transitions{job="prow-controller-manager",job_name=~".*-images",org="openshift-priv"}[6h])
```

Comment:
It is closed to what I suggested. :) 
Do we want to count e.g., aborted jobs?

In the example,

success rate = sum(rate(... {state="success"})) / (sum(rate(... {state="success"})) + sum(rate(... {state="failure"})))

Maybe reformat the alert with success rate instead of failure rate? Or define failure rate only counting `success` and `failure` jobs?

failure rate = sum(rate(... {state="failure"})) / (sum(rate(... {state="success"})) + sum(rate(... {state="failure"})))

Reply:
I would be ok including the aborted jobs as well. There are many reasons why the jobs is aborted and most of the reasons are legit